v1/src/test/resources/AvroToBigtableLT/schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "key": "server{{integer(1,50)}}",
+  "key": "server{{integer(1,500)}}",
   "cells": [{
       "family": "SystemMetrics",
       "qualifier": "{{random("cpu", "memory")}}",
```

Comment:
(not related to this PR) github UI suggests invalid json here

Reply:
That json file matches the format for Streaming Data Generator